packages/website/src/components/MentorPerspective.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+
+import { Button } from '@coderscamp/ui/components/Button';
+import { Center } from '@coderscamp/ui/components/Center';
+import { VStack } from '@coderscamp/ui/components/Stack';
+import { Typography } from '@coderscamp/ui/components/Typography';
+import { useBreakpointValue } from '@coderscamp/ui/hooks/useBreakpointValue';
+import { SolidDownloadIcon } from '@coderscamp/ui/icons';
+
+export const MentorPerspective = () => {
+  const downloadIcon = useBreakpointValue({ base: undefined, sm: <SolidDownloadIcon /> });
+
+  return (
+    <Center py="80px" backgroundColor="gray.100">
+      <VStack mx={{ base: '30px', xl: 'auto' }} maxW="min(1280px, 100%)">
```

Comment:
🤔 As you don't use the `spacing` prop here it would be better to replace VStack with the Flex

Reply:
Added spacing 🙃 with flex, I had to change flexDir to column and button had incorrect width, VStack looks better here